docs/fleet/fragments.md

```diff
@@ -28,95 +32,46 @@ Once you are confident that your configuration change works as expected, you can
-
-   There are three options for this:
-
-   - **Public:** Any user inside or outside of your organization will be able to view and use this fragment.
-   - **Private:** No user outside of your organization will be able to view or use this fragment.
-   - **Unlisted:** Any user inside or outside of your organization, with a direct link, will be able to view and use this fragment.
-
-4. Click **Save** to save your fragment.
-
-## Add a fragment to a machine
-
-To add a fragment to a machine:
-
-- Navigate to your machine in [the Viam app](https://app.viam.com/robots).
-- Go to the **CONFIGURE** tab. In the left-hand menu, click the **+** (Create) icon next to the machine {{< glossary_tooltip term_id="part" text="part" >}} you want to add the fragment to.
-- Select **Insert fragment**.
-  Now, you can see the available fragments to add:
-
-  {{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_list.png" resize="400x" style="max-width: 500px" alt="List of available fragments">}}
-
-- Look through the list of available fragments.
-  Click on a fragment to open its menu.
-  Click **View JSON** to view the JSON configuration the fragment contains, and click **Insert fragment** to insert the fragment into your machine's configuration.
-- Click **Save** in the upper right corner of the page to save your new configuration.
-
-The components and services included in the fragment will now appear as _read-only_ cards on the **CONFIGURE** tab, along with a card for your fragment:
-
-{{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_cards.png" resize="400x" style="max-width: 500px" alt="List of available fragments">}}
-
-In the `JSON` configuration, you will see the fragment ID in the `fragments` section:
-
-```json
-{
-  "components": [],
-  "fragments": ["3e8e0e1c-f515-4eac-8307-b6c9de7cfb84"]
-}
-```
-
-For an example of adding a fragment to a machine, see the [Add a Rover Fragment to your Machine](/appendix/try-viam/rover-resources/rover-tutorial-fragments/).
-
 ## Modify the config of a machine that uses a fragment
 
-The configuration cards of components and services included in the fragment are _read-only_, meaning they cannot be modified from the fragment specification.
-
-Likewise, when you modify the fragment itself, those changes are pushed to all machines that use that fragment.
+The configuration of components and services included in a fragment are _read-only_.
+Likewise, when you modify the fragment itself, any changes are pushed to all machines that use that fragment.
 
 If you need to modify the config of just one machine that uses a fragment, you have two options:
 
 - Use `fragment_mods` in your machine's config to overwrite certain fields of the fragment.
 - Copy and paste the contents of the fragment, remove the link to the fragment itself, then modify the config as needed.
   - If you use this method, future updates to the fragment _will not_ be automatically pushed to your machine.
```

Comment:
I think possibly this is not an option we should highlight. We only did this as a workaround I think. @nataliajacobowitz what do you think?